src/server/pegasus_server_impl.cpp

```diff
@@ -2894,5 +2916,39 @@ void pegasus_server_impl::set_partition_version(int32_t partition_version)
     // TODO(heyuchen): set filter _partition_version in further pr
 }
 
+::dsn::error_code pegasus_server_impl::check_meta_cf(const std::string &path,
+                                                     bool *need_open_with_meta_cf)
+{
+    *need_open_with_meta_cf = false;
+    std::vector<std::string> column_families;
+    auto s = rocksdb::DB::ListColumnFamilies(rocksdb::DBOptions(), path, &column_families);
+    if (!s.ok()) {
+        derror_replica("rocksdb::DB::ListColumnFamilies failed, error = {}", s.ToString());
+        return ::dsn::ERR_LOCAL_APP_FAILURE;
+    }
+
+    for (const auto &column_family : column_families) {
+        if (column_family == DATA_COLUMN_FAMILY_NAME) {
+            continue;
+        }
+        if (column_family == META_COLUMN_FAMILY_NAME) {
+            *need_open_with_meta_cf = true;
+            continue;
+        }
+        dassert_replica(true, "Column family '{}' should not present");
```

Comment:
```suggestion
        dassert_replica(false, "Column family '{}' should not present");
```

Reply:
data CF and meta CF have been checked above, this is a new CF we can't meet here

Reply:
Yes. But if you are sure that impossible to run into this statement, why not use `dassert(false`?
`dassert(true` is the same as a piece of comment. It makes reader wonder why you write this assert rather than simply a comment.

Reply:
I think I've made a mistake..